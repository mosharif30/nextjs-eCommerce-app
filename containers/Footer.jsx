import React from 'react'

/** @jsxImportSource @emotion/react */
import { css, useTheme } from '@emotion/react'

function Footer() {
  const theme = useTheme()

  return (
    <div>
      <navbar
        css={css`
          display: flex;
          background-color: ${theme.colors.primary};
          font-size: 24px;
          width: 100%;
          padding: 2%;
          justify-content: center;
          @media (max-width: 430px) {
            font-size: 18px;
          }
        `}
      >
        <span
          css={css`
            color: white;
          `}
        >
          ساخته شده توسط محمدرضا شریف خانی
        </span>
      </navbar>
    </div>
  )
}

export default Footer
