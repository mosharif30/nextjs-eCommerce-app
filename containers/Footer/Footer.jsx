import React from 'react'

/** @jsxImportSource @emotion/react */
import { css, useTheme } from '@emotion/react'
import { Small } from '../../components'

function Footer() {
  const theme = useTheme()

  return (
    <div>
      <div
        css={css`
          background-color: ${theme.colors.dark};
          font-size: 24px;
          width: 100%;
          padding: 2%;
          color: white;

          @media (max-width: 430px) {
            font-size: 18px;
          }
        `}
      >
        <div
          css={css`
            width: 80%;
            display: flex;
            flex-direction: column;
            margin-right: 10%;
          `}
        >
          <Small>ساخته شده توسط محمدرضا شریف خانی</Small>
          <Small>
            لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ، و با
            استفاده از طراحان گرافیک است، چاپگرها و متون بلکه روزنامه و مجله در
            ستون و سطرآنچنان که لازم است، و برای شرایط فعلی تکنولوژی مورد نیاز،
          </Small>
        </div>
      </div>
    </div>
  )
}

export default Footer
