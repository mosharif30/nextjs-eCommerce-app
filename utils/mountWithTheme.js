import { ThemeProvider } from '@emotion/react'
import theme from '../configs/theme'
const mountWithTheme 
