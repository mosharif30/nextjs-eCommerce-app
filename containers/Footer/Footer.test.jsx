import mountWithTheme from '../../utils/mountWithTheme'
import Footer from './Footer'
describe('Footer', () => {
  test('render correctly', () => {
    const component = mountWithTheme(<Footer></Footer>)
    const tree = component
    expect(tree).toMatchSnapshot()
  })
})
