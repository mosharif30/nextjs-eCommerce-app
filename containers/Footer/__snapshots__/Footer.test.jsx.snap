// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render correctly 1`] = `
<div>
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-yuk4ah-Footer{background-color:#34495e;font-size:24px;width:100%;padding:2%;color:white;}@media (max-width: 430px){.css-yuk4ah-Footer{font-size:18px;}}",
      }
    }
    data-emotion="css yuk4ah-Footer"
  />
  <div
    className="css-yuk4ah-Footer"
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-1lox1gh-Footer{width:80%;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;margin-right:10%;}",
        }
      }
      data-emotion="css 1lox1gh-Footer"
    />
    <div
      className="css-1lox1gh-Footer"
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-194r2pk-Small{margin:3rem 0 1.38rem;text-align:right;font-weight:400;line-height:1.3;margin-top:0;font-size:0.889em;}",
          }
        }
        data-emotion="css 194r2pk-Small"
      />
      <h1
        className="css-194r2pk-Small"
      >
        ساخته شده توسط محمدرضا شریف خانی
      </h1>
      <h1
        className="css-194r2pk-Small"
      >
        لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ، و با استفاده از طراحان گرافیک است، چاپگرها و متون بلکه روزنامه و مجله در ستون و سطرآنچنان که لازم است، و برای شرایط فعلی تکنولوژی مورد نیاز،
      </h1>
    </div>
  </div>
</div>
`;
